### ISSUE TITLE
no pip3 package available

### ISSUE THREAD

**Issue by buanzo** [gertvdijk/purepythonmilter#20]
Hi! I'm coding a cool milter for postfix, and ended up switching between pymilter, libmilter and now purepythonmilter. So far I like it, but the library cant be installed with pip3 install purepythonmilter as it says it is not found.

I will try installing from source, but I'd like to know if I can help somehow?

**Comment by wbolster** [comment]
can you share more details? in a fresh python 3.12 venv, it works fine for me:

``` bash
$ pip install purepythonmilter
Collecting purepythonmilter
  Using cached purepythonmilter-0.0.1-py3-none-any.whl.metadata (14 kB)
Collecting attrs (from purepythonmilter)
  Using cached attrs-23.2.0-py3-none-any.whl.metadata (9.5 kB)
Using cached purepythonmilter-0.0.1-py3-none-any.whl (49 kB)
Using cached attrs-23.2.0-py3-none-any.whl (60 kB)
Installing collected packages: attrs, purepythonmilter
Successfully installed attrs-23.2.0 purepythonmilter-0.0.1
```

**Comment by buanzo** [comment]
Indeed, the problem was I am on python 3.9. 

It is available from 3.10+ - Regrettably I cannot yet update the server that needs this tool to 3.10 or above. For that reason I am forced to drop this beautiful library and go to "import Milter"... 

Hopefully this situation changes in the near future.

Keep up the great work!
